feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -301,3 +279,15 @@ private fun CameraUseCase.LensFacing.toCameraSelector() = when(this) {
     CameraUseCase.LensFacing.FRONT -> CameraSelector.LENS_FACING_FRONT
     CameraUseCase.LensFacing.BACK -> CameraSelector.LENS_FACING_BACK
 }
+
+private fun CameraUseCase.FlashMode.toUiState() = when(this) {
+    CameraUseCase.FlashMode.OFF -> FlashModeStatus.OFF
```

Comment:
Again, probably not for this PR, but this is pointing out a lot of boilerplate for converting between shadowed enums. Can we just move all of these enums into a common `model` module that is used by `domain`, `data`, and `feature` modules?